share_app/src/main/java/org/odk/share/database/ShareDatabaseHelper.java

```diff
@@ -0,0 +1,76 @@
+package org.odk.share.database;
+
+import android.content.ContentValues;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteOpenHelper;
+
+import org.odk.share.application.Share;
+
+import timber.log.Timber;
+
+import static org.odk.share.dto.TransferInstance.ID;
+import static org.odk.share.dto.TransferInstance.INSTANCE_ID;
+import static org.odk.share.dto.TransferInstance.INSTRUCTIONS;
+import static org.odk.share.dto.TransferInstance.LAST_STATUS_CHANGE_DATE;
+import static org.odk.share.dto.TransferInstance.REVIEWED;
+import static org.odk.share.dto.TransferInstance.TRANSFER_STATUS;
+
+/**
+ * Created by laksh on 6/13/2018.
+ */
+
+public class ShareDatabaseHelper extends SQLiteOpenHelper {
+
+    private static final String DATABASE_NAME = "share.db";
+    private static final String SHARE_TABLE_NAME = "share";
```

Comment:
Are you planning to store status info about instances and forms in the same table or different ones? Is `share` the best table name?

Reply:
I'll be storing all instances transfer and feedback details in the same table, I'm not good at naming things, that's why I named it as `share`. Can you suggest some other name?

Reply:
😊Naming is hard. Maybe something like `transfer-metadata` or `transfer-summary` or even just `transfers`?

Reply:
I just realized that you named your DTO well so that will bring things nicely in line. 👍 https://twitter.com/timbray/status/817025379109990402?cn=cmVwbHk%3D 😄